.github/workflows/build-and-push-docker-image.yaml

```diff
@@ -0,0 +1,39 @@
+name: Build and push Docker image
```

Comment:
Sync this file with https://github.com/schul-cloud/schulcloud-synapse-synchronization/blob/develop/.github/workflows/build-and-push-docker-image.yaml

Reply:
Change to the version from schulcloud-synapse-synchronization.